package/vmalert.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Alerting for high-performance, cost-effective and scalable time series database, long-term remote storage for Prometheus.
+After=network.target
+
+[Service]
+Type=simple
+StartLimitBurst=5
+StartLimitInterval=0
+Restart=on-failure
+RestartSec=1
+ExecStart=/usr/bin/vmalert-prod -datasource.url=http://localhost:8428 -notifier.url=http://127.0.0.1:9093 -remoteRead.url=http://127.0.0.1:8428 -remoteWrite.url=http://127.0.0.1:8428 -rules=/etc/prometheus/rules.yaml
```

Comment:
Launch string looks weird. Should be probably moved to /etc/defautls?

Reply:
Looks like it's ok. But it will be better to change /usr/bin to /usr/local/bin. 
For datasorce is a good practice to use configuration file .ini, .toml, .yml where user can configure sever address for it's own instead localhost.

Reply:
> Launch string looks weird. Should be probably moved to /etc/defautls?

I have planned to override systemd unit in system where this deb pacakge will be used however it will be better to move launch string to `/etc/defaults`. I vaguely remember reading somewhere that overriding systemd unit using `/etc/systemd/system/<unit_name>.d` is preferred to using `/etc/defaults` but cannot find it and personally I like `/etc/defaults` more :)

> But it will be better to change /usr/bin to /usr/local/bin.

Even though this deb package is not up to debian standards deb package should install binaries to `/usr/bin`; `/usr/local/bin` should be free to use for system administrator. Given that original `victoria-metrics` pacakge used `/usr/bin` and it is the debian way I opted to keep it there.

> For datasource is a good practice to use configuration file .ini, .toml, .yml where user can configure server address for it's own instead localhost.

I am not aware that `vmalert` accepts such config file. Does it?

Reply:
i totally understand it, and I'm very sorry if my comments sounded rude or harsh. I'll do my best to contribute later to the packaging. i'm already supposed to push the freebsd package of VM.

NB: thanks for having done the packaging. My last company used, and is still using your packaging scripts !

Reply:
> i totally understand it, and I'm very sorry if my comments sounded rude or harsh

Not in the least. I hope I do not sound like a lazy uncaring person.

>  I'll do my best to contribute later to the packaging. i'm already supposed to push the freebsd package of VM.

I would like to do the same, currently circumstances do not allow me to invest more time in this. I will use the packages as they are. I'd like to refactor the packages to use debhelpers and be "Debian".

> NB: thanks for having done the packaging. My last company used, and is still using your packaging scripts !

Actually the original packages are not my work. Still it is good to hear they are useful even as they are. Gives this PR a reason to live.